VersionInfo.cpp

```diff
@@ -192,6 +195,33 @@ static const char * get_build_date()
 }
 #endif
 
+#if (defined(UNITEX_VERSION_BUILD_YEAR) &&\
+        defined(UNITEX_VERSION_BUILD_MONTH) &&\
+        defined(UNITEX_VERSION_BUILD_DAY))
+
+const char buildDateYYYYMMDD[] =
+{
+  UNITEX_VERSION_BUILD_YEAR[0], UNITEX_VERSION_BUILD_YEAR[1],
+  UNITEX_VERSION_BUILD_YEAR[2], UNITEX_VERSION_BUILD_YEAR[3],
+  '-',
+  UNITEX_VERSION_BUILD_MONTH[0], UNITEX_VERSION_BUILD_MONTH[1],
+  '-',
+  UNITEX_VERSION_BUILD_DAY[0], UNITEX_VERSION_BUILD_DAY[1],
+  '\0'
+};
+
+static const char * get_build_dateYYYYMMDD()
```

Comment:
[The open curly brace is always on the end of the last line of the function declaration, not the start of the next line.](https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions)